### PR TITLE
AutoDockSuite-4.2.5.1-goolf-1.4.10.eb

### PR DESCRIPTION
**Pull request by pescobar**

### CODE REVIEW

**Comment by fgeorgatos** [comment]
Test report by @fgeorgatos
**FAILED**
Build succeeded for 0 out of 1
Linux debian 6.0.10, Intel(R) Xeon(R) CPU           L5640  @ 2.27GHz, Python 2.6.6
See https://gist.github.com/f99ef29d98cc44dc4486 for a full test report.

File: easybuild/easyconfigs/a/AutoDockSuite/AutoDockSuite-4.2.5.1-goolf-1.4.10.eb [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+ autodock performs the docking of the ligand to a set of grids describing the target protein; 
+ autogrid pre-calculates these grids."""
+
+toolchain = {'name': 'goolf', 'version': '1.4.10'}
+
+source_urls = ['http://autodock.scripps.edu/downloads/autodock-registration/tars/dist4251/']
+sources = ['%(namelower)s-%(version)s-src.tar.gz']
+
+preconfigopts = ["cd autodock && ", "cd autogrid && "]
+premakeopts = ["cd autodock && ", "cd autogrid && "]
```
Comment (fgeorgatos) [inline easybuild/easyconfigs/a/AutoDockSuite/AutoDockSuite-4.2.5.1-goolf-1.4.10.eb]: prebuildopts; otherwise, this looks good!

**Comment by fgeorgatos** [comment]
minor changes needed here, `easyblock = 'ConfigureMake'` ,put this in REVIEW plz!

**Comment by migueldiascosta** [comment]
@pescobar, this PR is being closed for the following reason(s): inactive.
Please don't hesitate to reopen this PR or add a comment if you feel this contribution is still relevant.
For more information on our policy w.r.t. closing PRs, see https://easybuild.readthedocs.io/en/latest/Contributing.html#why-a-pull-request-may-be-closed-by-a-maintainer

**Pull request closed without merging**